### PR TITLE
Changing the internal_listener related extensions to stable.

### PR DESCRIPTION
**Pull request by yanjunxiang-google**
Internal listener extensions had been there for a while. It's time to change them to stable.

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: source/extensions/extensions_metadata.yaml [hunk 1]
```diff
@@ -1155,8 +1155,8 @@ envoy.transport_sockets.alts:
 envoy.transport_sockets.internal_upstream:
   categories:
   - envoy.transport_sockets.upstream
-  security_posture: unknown
-  status: alpha
+  security_posture:  robust_to_untrusted_downstream_and_upstream
+  status: stable
```
Comment (kyessenov) [inline source/extensions/extensions_metadata.yaml:1159]: Can you also change the protobuf for the registry and remove "WIP" warning?

**Review by adisuissa — COMMENTED** [review]
/lgtm api

File: source/extensions/extensions_metadata.yaml (hunk 1 above)
Comment (yanjunxiang-google) [reply source/extensions/extensions_metadata.yaml:1159]: done

**Review by yanavlasov — APPROVED** [review]

**Pull request merged**